lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -2520,11 +2520,41 @@ DefConstructorI('\hidden@align', undef, "",   alias => '');
 
 # Handled directly in alignments, but must be defined as non-macros
 DefPrimitiveI('\noalign', undef, sub {
-    Error('unexpected', '\noalign', $_[0], "\\noalign cannot be used here"); });
+    $_[0]->bgroup;
+    Error('unexpected', '\noalign', $_[0], "\\noalign cannot be used here");
+    $STATE->assignMeaning(T_ALIGN,
+      $STATE->lookupMeaning(T_CS('\relax')), 'local');
+    $STATE->assignMeaning(T_CS('\noalign'),
+      $STATE->lookupMeaning(T_CS('\relax')), 'local');
+    $STATE->assignMeaning(T_CS('\omit'),
+      $STATE->lookupMeaning(T_CS('\relax')), 'local');
+    $STATE->assignMeaning(T_CS('\span'),
+      $STATE->lookupMeaning(T_CS('\relax')), 'local');
```

Comment:
Wouldn't `Let` be clearer (and significantly less scary?)

Reply:
It would, done!